guides/common/modules/proc_adding-hosts-to-a-host-collection-in-bulk-by-using-cli.adoc

```diff
@@ -0,0 +1,29 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="adding-hosts-to-a-host-collection-in-bulk-by-using-cli"]
+= Adding hosts to a host collection in bulk by using Hammer CLI
+
+[role="_abstract"]
+You can add multiple hosts to a host collection by using Hammer CLI.
+
+.Prerequisites
+ifdef::satellite[]
+A host must be registered to {ProjectName} to add it to a host collection.
+For more information about registering hosts, see xref:Registering_Hosts_by_Using_Global_Registration_{context}[].
+endif::[]
+
+ifndef::satellite[]
+A host must be registered to {ProjectName} to add it to a host collection.
+endif::[]
+
+Note that if you add a host to a host collection, the {Project} auditing system does not log the change.
```

Comment:
```suggestion
Note that if you add hosts to a host collection, the {Project} auditing system does not log the change.
```

suggestion: using plural would align the wording with the title. same probably applies to the WebUI module.

Reply:
I don't think this is necessary. I've seen this elsewhere too -- a heading would use plural (because it's general) while this describes a specific situation a user is in (after just adding a host). No change.